parsec/_parsec_pyi/backend_connection.pyi

```diff
@@ -1,10 +1,196 @@
+from __future__ import annotations
+
+from typing import Union
+
 from oxidation.bindings.generator.api import DeviceID
 from parsec._parsec_pyi.addrs import BackendOrganizationAddr
 from parsec._parsec_pyi.crypto import SigningKey
-from parsec._parsec_pyi.protocol.ping import AuthenticatedPingRep
+from parsec._parsec_pyi.protocol import (
+    AuthenticatedPingReq,
+    BlockCreateReq,
+    BlockReadReq,
+    DeviceCreateReq,
+    EventsListenReq,
+    EventsSubscribeReq,
+    HumanFindReq,
+    Invite1GreeterWaitPeerReq,
+    Invite2aGreeterGetHashedNonceReq,
+    Invite2bGreeterSendNonceReq,
+    Invite3aGreeterWaitPeerTrustReq,
+    Invite3bGreeterSignifyTrustReq,
+    Invite4GreeterCommunicateReq,
+    InviteDeleteReq,
+    InviteListReq,
+    InviteNewReq,
+    MessageGetReq,
+    OrganizationConfigReq,
+    OrganizationStatsReq,
+    PkiEnrollmentAcceptReq,
+    PkiEnrollmentListReq,
+    PkiEnrollmentRejectReq,
+    RealmCreateReq,
+    RealmFinishReencryptionMaintenanceReq,
+    RealmGetRoleCertificatesReq,
+    RealmStartReencryptionMaintenanceReq,
+    RealmStatsReq,
+    RealmStatusReq,
+    RealmUpdateRolesReq,
+    UserCreateReq,
+    UserGetReq,
+    UserRevokeReq,
+    VlobCreateReq,
+    VlobListVersionsReq,
+    VlobMaintenanceGetReencryptionBatchReq,
+    VlobMaintenanceSaveReencryptionBatchReq,
+    VlobPollChangesReq,
+    VlobReadReq,
+    VlobUpdateReq,
+)
+
+AllAuthenticatedCmds = Union[
+    BlockCreateReq,
+    BlockReadReq,
+    DeviceCreateReq,
+    EventsListenReq,
+    EventsSubscribeReq,
+    HumanFindReq,
+    Invite1GreeterWaitPeerReq,
+    Invite2aGreeterGetHashedNonceReq,
+    Invite2bGreeterSendNonceReq,
+    Invite3aGreeterWaitPeerTrustReq,
+    Invite3bGreeterSignifyTrustReq,
+    Invite4GreeterCommunicateReq,
+    InviteDeleteReq,
+    InviteListReq,
+    InviteNewReq,
+    MessageGetReq,
+    OrganizationStatsReq,
+    PkiEnrollmentAcceptReq,
+    PkiEnrollmentListReq,
+    PkiEnrollmentRejectReq,
+    RealmCreateReq,
+    RealmFinishReencryptionMaintenanceReq,
+    RealmGetRoleCertificatesReq,
+    RealmStartReencryptionMaintenanceReq,
+    RealmStatsReq,
+    RealmStatusReq,
+    RealmUpdateRolesReq,
+    UserCreateReq,
+    UserGetReq,
+    UserRevokeReq,
+    VlobCreateReq,
+    VlobListVersionsReq,
+    VlobMaintenanceGetReencryptionBatchReq,
+    VlobMaintenanceSaveReencryptionBatchReq,
+    VlobPollChangesReq,
+    VlobReadReq,
+    VlobUpdateReq,
+    OrganizationConfigReq,
+    AuthenticatedPingReq,
+]
+
+class AuthenticatedCmdsType:
+    @classmethod
+    def BLOCK_CREATE(cls, req: BlockCreateReq) -> AuthenticatedCmdsType: ...
```

Comment:
BUT WHY THE SCREAMING CASE FOR ? DON'T WE WANT A BIT OF PEACEFULNESS IN THIS WORLD ?